aws_cloudwatch.go

```diff
@@ -81,20 +136,16 @@ func createGetMetricStatisticsInput(dimensions []*cloudwatch.Dimension, namespac
 		ExtendedStatistics: extendedStatistics,
 	}
 
-	if *debug {
-		if len(statistics) != 0 {
-			log.Println("CLI helper - " +
-				"aws cloudwatch get-metric-statistics" +
-				" --metric-name " + metric.Name +
-				" --dimensions " + dimensionsToCliString(dimensions) +
-				" --namespace " + *namespace +
-				" --statistics " + *statistics[0] +
-				" --period " + strconv.FormatInt(period, 10) +
-				" --start-time " + startTime.Format(time.RFC3339) +
-				" --end-time " + endTime.Format(time.RFC3339))
-		}
-		log.Println(*output)
-	}
+	log.Println("CLI helper - " +
```

Comment:
why change this?

Reply:
I was missing some log saying about the metric requests, for some reason this log was not being shown even with the flag enabled. :disappointed:

Reply:
Sample line:
```
`2019/10/18 18:13:30 CLI helper - aws cloudwatch get-metric-statistics --metric-name NumberOfMessagesReceived --dimensions Name=QueueName,Value=ProdInsiderCreateReviewDLQ --namespace AWS/SQS --statistics Sum --period 600 --start-time 2019-10-18T18:03:30Z --end-time 2019-10-18T18:13:30Z
`
```